test-data/unit/stubgen.test

```diff
@@ -1544,6 +1544,15 @@ from typing import TypeVar
 T = TypeVar('T')
 alias = Union[T, List[T]]
 
+[case testTypeAlias]
+from typing import TypeAlias
+
+alias: TypeAlias = tuple[int, str]
+
+[out]
+alias = tuple[int, str]
```

Comment:
I don't think that we should skip `TypeAlias` here. It should be like this, in my opinion:

```suggestion
from typing import TypeAlias

alias: TypeAlias = tuple[int, str]
```